objects/tag.c

```diff
@@ -91,6 +91,83 @@
  *
  *    local t = awful.tag.find_by_name(awful.screen.focused(), "name")
  *
+ * **Common shortcuts**:
+ *
+ * Here is a few useful shortcuts not part of the default `rc.lua`. Add these
+ * functions above `-- {{{ Key bindings`:
+ *
+ * Delete the current tag
+ *
+ *    local function delete_tag()
+ *        local t = awful.screen.focused().selected_tag
+ *        if not t then return end
+ *        t:delete()
+ *    end
+ *
+ * Create a new tag at the end of the list
+ *
+ *    local function add_tag()
+ *        awful.tag.add("NewTag",{screen= awful.screen.focused() }):view_only()
+ *    end
+ *
+ * Rename the current tag
+ *
+ *    local function rename_tag()
+ *        awful.prompt.run {
+ *            prompt       = "New tag name: ",
+ *            textbox      = awful.screen.focused().mypromptbox.widget,
+ *            exe_callback = function(new_name)
+ *                if not new_name or #new_name == 0 then return end
+ *
+ *                local t = awful.screen.focused().selected_tag
+ *                if t then
+ *                    t.name = new_name
+ *                end
+ *            end
+ *        }
+ *    end
+ *
+ * Move the focused client to a new tag
+ *
+ *    local function move_to_new_tag()
+ *        local c = client.focus
+ *        if not c then return add_tag() end
```

Comment:
Should this really call `add_tag()` here? I would expect this to do nothing if no key is focused.